packages/plugin-apex/test/commands/force/apex/test/run.test.ts

```diff
@@ -125,7 +125,7 @@ describe('force:apex:test:run', () => {
     .it('should handle a tap format parsing error', ctx => {
       expect(ctx.stdout).to.contain('{\n  "tests": []\n}\n');
       expect(ctx.stderr).to.contain(
-        messages.getMessage('testResultProcessErr', ['TypeError: '])
+        messages.getMessage('testResultProcessErr', [''])
```

Comment:
Do you know why this test needed to be updated?

Reply:
I am not sure. I suspect bumping typescript version had something to do with this.